src/components/datepicker/DatePickerControls.vue

```diff
@@ -6,7 +6,10 @@
       class="datepicker-controls__prev"
       @click="changeVisibleDate('prev')"
     >
-      <svg viewBox="0 0 24 24">
+      <svg v-if="rtl" viewBox="0 0 24 24">
+        <path d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"></path>
+      </svg>
+      <svg v-else viewBox="0 0 24 24">
```

Comment:
Why do you inverse arrow ?

Reply:
Because the arrows are not pointing the right direction otherwise. You can set `dir="rtl"` on the HTML element to see it for yourself. They are doing the correct action of going previous and next, but the icon is flipped.

Reply:
Oh 😮 I understood. There is another way to do it without this. I think I’ll lerge your code and update it.